client/src/site-search/index.scss

```diff
@@ -1,6 +1,6 @@
 .site-search {
   margin: 20px;
-  min-height: 1000px;
+  min-height: 1200px;
```

Comment:
Is there a reason why we use `px` values here?
How does it behave on Zoom? <kbd>Ctrl</kbd>+<kbd>+</kbd> for example.

Reply:
Excellent point! 
But the idea is that I'll build the basic functionality and the @schalkneethling corrects/perfects all the SCSS and stuff. 
But pointing this out will help me when I "turn it over" to him.